example/lib/main.dart

```diff
@@ -0,0 +1,117 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_eco_mode/flutter_eco_mode.dart';
+
+void main() {
+  runApp(MyApp());
+}
+
+class MyApp extends StatefulWidget {
+  MyApp({super.key});
+
+  final flutterEcoModePlugin = FlutterEcoMode();
+
+  @override
+  State<MyApp> createState() => _MyAppState();
+}
+
+class _MyAppState extends State<MyApp> {
+  String? _platformInfo;
+
+  @override
+  void initState() {
+    super.initState();
+    _getPlatformInfo();
+  }
+
+  void _getPlatformInfo() async {
+    final platformInfo = await widget.flutterEcoModePlugin.getPlatformInfo();
+    setState(() {
+      _platformInfo = platformInfo;
+    });
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      home: Scaffold(
+        appBar: AppBar(
+          title: const Text('Plugin example app'),
+        ),
+        body: Center(
+          child: Column(
+            children: [
+              Text('Platform info: $_platformInfo\n'),
+              ElevatedButton(
+                onPressed: () async {
+                  final data =
+                      await widget.flutterEcoModePlugin.getBatteryLevel();
+                  print(data);
```

Comment:
Maybe we should see all the result directly in the screen ? instead of print in the console ?

Reply:
Will be released with this issue: #8